src/features/DisplayControls/NursingTasks/components/TaskTile.jsx

```diff
@@ -7,11 +7,15 @@ import data from "../../../../utils/config.json";
 import { Tag, TooltipDefinition } from "carbon-components-react";
 
 import "../styles/TaskTile.scss";
+import { getTagForTheDrugOrder } from "../../../../utils/DisplayTags";
+import { useFetchIpdConfig } from "../../../../entries/Dashboard/hooks/useFetchIpdConfig";
 import { FormattedMessage } from "react-intl";
 
 export default function TaskTile(props) {
   const { medicationNursingTask } = props;
   const newMedicationNursingTask = medicationNursingTask[0];
+  const { configData } = useFetchIpdConfig();
```

Comment:
we are fetching this config.json or making this API call at multiple components, can we do it via context API so that it can be shared across multiple components and make a single API call?

Reply:
Changes are made. Please review again